### PR TITLE
Reafactor/update daos

### PR DESCRIPTION
**Pull request by OropezaHugo**
updating daos
connect daos with some services
refactor user and contact posts

### CODE REVIEW

File: backend/Converters/ToPostDTO/ContactPostConverter.cs [hunk 1]
```diff
@@ -5,20 +5,15 @@
...
 
         return new ContactPostDTO
         {
             PhoneNumber = contact.PhoneNumber,
-            Email = contact.Email,
-            ReservationList = reservationList
+            Email = contact.Email
         };
     }
 }
```
Comment (MayerliSantander) [inline backend/Converters/ToPostDTO/ContactPostConverter.cs]: Add end line

File: backend/DB/Operations/Concrete/RoombathInformationDAO.cs [hunk 2]
```diff
@@ -200,4 +200,24 @@ public List<RoomBathInformation> GetRoombathInformationsByRoomTemplateId(Guid ro
...
+        var reader = com.ExecuteReader();
+        int recordsAffected;
+
+        reader.Read();
+        recordsAffected = reader.RecordsAffected;
+        reader.Close();
+
+        return recordsAffected>0;
+    }
 }
```
Comment (MayerliSantander) [inline backend/DB/Operations/Concrete/RoombathInformationDAO.cs]: Add end line

File: backend/Services/RoomFiltersService.cs [hunk 3]
```diff
@@ -137,4 +150,4 @@ public async Task<bool> IsAvailable(Room room, AvailabilityRequestDTO availabili
     }
 
 
-}
+}
```
Comment (MayerliSantander) [inline backend/Services/RoomFiltersService.cs]: Add endline

File: backend/Services/RoomTemplateService.cs [hunk 4]
```diff
@@ -110,4 +121,4 @@ public async Task<RoomTemplatePostDTO> UpdateElementById(Guid roomTemplateId, Ro
 
         return roomTemplatePostDto;
     }
-}
+}
```
Comment (MayerliSantander) [inline backend/Services/RoomTemplateService.cs]: Add endline

**Review by MayerliSantander — CHANGES_REQUESTED** [review]

**Review by MayerliSantander — APPROVED** [review]

**Review by JCarlosHidalgoSosa — APPROVED** [review]

**Review by DeidamiaFuentes — APPROVED** [review]

**Pull request merged**